server/grpcsvr/taskrunner/taskrunner.go

```diff
@@ -11,9 +11,7 @@ import (
 	"github.com/pkg/errors"
 
```

Comment:
should drop this extra new-line

Reply:
you sure? this is how `go fmt` and `goimports` formatted this file.